subprojects/core/src/main/java/org/gradle/api/internal/tasks/DefaultTaskInputFilePropertyRegistration.java

```diff
@@ -33,7 +35,7 @@ public class DefaultTaskInputFilePropertyRegistration extends AbstractTaskFilePr
     private boolean skipWhenEmpty;
     private DirectorySensitivity directorySensitivity = DirectorySensitivity.DEFAULT;
     private LineEndingSensitivity lineEndingSensitivity = LineEndingSensitivity.DEFAULT;
-    private Class<? extends FileNormalizer> normalizer = AbsolutePathInputNormalizer.class;
+    private Normalizer normalizer = InputNormalizer.ABSOLUTE_PATH;
```

Comment:
Normalizer seems like a generic name. Could we use something more specific?

Reply:
Let's rename it to `FileNormalizer` for now.

Reply:
Handled in https://github.com/gradle/gradle/pull/22759.